roles-in-tech/security.md

```diff
@@ -90,7 +90,9 @@ These are some misconceptions that people have in their mind but it is not alway
 - **Small organizations will not be attacked** - Major organizations assume that they are small and will not be targeted as we are small company. The research shows that 70% of cyber attacks target small businesses. So today for everyone it is very important to take care of its security if its data is worth value for money, whether it is a website for fun or it is a huge organization.
 
 ### Guided Practice
-- Talk to as many professionals as you can [National Cyber Security Centre](https://live.ncsc.gov.uk/topics/new-talent) is worth for who lives in UK
+- *Talk to as many professionals as you can*
+
+ [National Cyber Security Centre](https://live.ncsc.gov.uk/topics/new-talent) is worth for who lives in UK
```

Comment:
can you describe this link a little more here?  All I know by looking at it is that it probably happens in the UK.